Building simpll-lib as shared fails

Running:
```
mkdir build && cd build
cmake .. -GNinja -DBUILD_SHARED_LIBS=On
ninja
```

fails with linker errors:
```
Linking CXX executable diffkemp/simpll/diffkemp-simpll
FAILED: diffkemp/simpll/diffkemp-simpll 
: && /usr/bin/c++ -fno-rtti -fpic -fvisibility=hidden -Wall -Wextra  diffkemp/simpll/CMakeFiles/simpll.dir/SimpLL.cpp.o -o diffkemp/simpll/diffkemp-simpll  -Wl,-rpath,/home/vmalik/src/diffkemp/diffkemp/build-test/diffkemp/simpll:  diffkemp/simpll/libsimpll-lib.so  -lLLVM-11 && :
/usr/bin/ld: diffkemp/simpll/CMakeFiles/simpll.dir/SimpLL.cpp.o: in function `main':
SimpLL.cpp:(.text+0x6d): undefined reference to `Config::Config()'
/usr/bin/ld: SimpLL.cpp:(.text+0x95): undefined reference to `processAndCompare(Config&, OverallResult&)'
/usr/bin/ld: SimpLL.cpp:(.text+0xae): undefined reference to `reportOutput(Config&, OverallResult&)'
collect2: error: ld returned 1 exit status
```

--- a/diffkemp/simpll/CMakeLists.txt
+++ b/diffkemp/simpll/CMakeLists.txt
@@ -19,7 +19,7 @@ file(GLOB passes passes/*.cpp)
 file(GLOB library library/*.cpp)
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti -fpic -fvisibility=hidden -Wall -Wextra")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti -fpic -Wall -Wextra")
 
 exec_program(llvm-config ARGS --libs irreader passes support OUTPUT_VARIABLE llvm_libs)
 exec_program(llvm-config ARGS --system-libs OUTPUT_VARIABLE system_libs)

